src/commands/ext-configure.ts

```diff
@@ -34,6 +34,7 @@ marked.setOptions({
 export default new Command("ext:configure <extensionInstanceId>")
   .description("configure an existing extension instance")
   .withForce()
+  .option("--local", "deprecated")
```

Comment:
Strongly, strongly consider printing a warning if `--local` is included. IIRC, you should be able to check `options.local`.

Reply:
Good point - done